hpccm/building_blocks/gdrcopy.py

```diff
@@ -83,16 +84,8 @@ def __init__(self, **kwargs):
         self.__ospackages = kwargs.pop('ospackages', ['make', 'wget'])
         self.__prefix = kwargs.pop('prefix', '/usr/local/gdrcopy')
         self.__toolchain = kwargs.pop('toolchain', toolchain())
-        self.__version = kwargs.pop('version', '2.1')
+        self.__version = kwargs.pop('version', '2.2')
```

Comment:
Can you please update the docstring to reflect the version bump?

If you don't have `pydocmd`, I can regenerate the docs myself.